example/lib/movas/services/posts/posts_service.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:movas_example/movas/services/responses/feed_items.dart';
+import 'package:rxdart/rxdart.dart';
+
+abstract class PostsService {
+  PostsService(this.feedItemsResponse$);
+
+  final PublishSubject<FeedItemsResponse> feedItemsResponse$;
+
+  Future<void> getPosts();
+
+  void emptyItems() {
```

Comment:
no need to call empty on the service. service just publishes updates when getting called. the data is saved in store, and you should empty the store.